### ISSUE TITLE
Term name and definition update: collagen sheet (GO:0098646) to interstitial hexagonal collagen network

### ISSUE THREAD

**Issue by DaiqingChen** [geneontology/go-ontology#29415]
* **GO term ID and label for which you request a definition update**
Collagen sheet (GO:0098646)
Note: "collagen sheet" is NOT synonyms of interstitial hexagonal collagen network, requesting "definition update" instead of obsoleting and adding new term because the annotation refers to the same gene product (collagen type X trimer (GO:0005599)).

* **New proposed definition (genus-differentia pattern, plus additional information if needed)**
- [x] A hexagonal-shaped collagen network formed by type X collagen trimers, mostly found in the cartilage hyaline matrix.

* **Reference(s)**
Type X collagen trimers forms hexagonal supramolecular structure: PMID: 21421911
Type X collagen trimers occurs in the interstitial matrix of cartilage: PMID: 34948124, PMID: 9151143

* **Parent term(s)**
- [ ] Collagenous component of interstitial matrix (NTR #29473) 
- [x] Collagen network(GO:0098645)

* **Children terms (if applicable)**
Collagen type X trimer (GO:0005599)

* Remove gene product 
- [ ] remove COL8A1 (UniProtKB:P27658, 9606 Homo sapiens) from the annotation

* **Additional information**
The network-forming collagens can form two types of supramolecular structures (chicken-wire and hexagonal), the hexagonal collagen network is formed by type VIII collagen trimers in the basement membrane and by type X collagen trimers in the interstitial matrix. 

**See slides 1, 3, and 7:** https://docs.google.com/presentation/d/1v4o53iJN2Ywo1qi3L8ANZg-p6uCSVaGi/edit?usp=sharing&ouid=109267399050361021326&rtpof=true&sd=true

---

**Slide screenshots:** 

<img width="1482" alt="Image" src="https://github.com/user-attachments/assets/8ab87788-611f-475c-88d6-597b37857416" />

---

<img width="1667" alt="Image" src="https://github.com/user-attachments/assets/f9588c3f-1b2e-4011-9b39-8499e06e200a" />

---

<img width="1485" alt="Image" src="https://github.com/user-attachments/assets/59c53cdc-d480-44bc-a627-b27ad6c4a5c6" />

**Comment by pgaudet** [comment]
collagen type X trimer (GO:0005599) is already part_of collagen sheet (GO:0098646), is this OK ?

**Comment by pgaudet** [comment]
Hi @DaiqingChen 

I dont see  COL8A1(UniProtKB:P27658, 9606 Homo sapiens annotated to collagen sheet (GO:0098646); which annotation do you want to change? 

<img width="445" alt="image" src="https://github.com/user-attachments/assets/7ca921fe-4875-4953-ac7c-63875cfc492e" />


Thanks, Pascale

**Comment by pgaudet** [comment]
Term now looks like this: 

> [Term]
id: GO:0098646
name: interstitial hexagonal collagen network
namespace: cellular_component
def: "A hexagonal shaped collagen network formed by collagen type X trimer, mostly found in the cartilage hyaline matrix, which located in the interstitial area." [PMID:21421911, PMID:34948124]
synonym: "collagen hexagonal network" EXACT [PMID:21421911]
synonym: "collagen sheet" EXACT []
is_a: GO:0098645 ! collagen network

**Comment by DaiqingChen** [comment]
> collagen type X trimer (GO:0005599) is already part_of collagen sheet (GO:0098646), is this OK ? (I dont think it should be is_a? )

Yes, collagen type X trimer(GO:0005599) should be part_of collagen sheet(GO:0098646).

**Comment by DaiqingChen** [comment]
> Hi [@DaiqingChen](https://github.com/DaiqingChen)
> 
> I dont see COL8A1(UniProtKB:P27658, 9606 Homo sapiens annotated to collagen sheet (GO:0098646); which annotation do you want to change?
> 
> <img alt="image" width="445" src="https://private-user-images.githubusercontent.com/4782928/395480564-7ca921fe-4875-4953-ac7c-63875cfc492e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQxMDY3NDksIm5iZiI6MTczNDEwNjQ0OSwicGF0aCI6Ii80NzgyOTI4LzM5NTQ4MDU2NC03Y2E5MjFmZS00ODc1LTQ5NTMtYWM3Yy02Mzg3NWNmYzQ5MmUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTNUMTYxNDA5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODM5ZGYxNDRiNDUxZjRjYzY2Mjc1NmNjMGNlNjE4NDhlYjI3Y2I3ODE0YTdjNDRhNmJhM2M3MDBiOTNhODQ1YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.OYDQuJSnA5S5-np5AC34tA2vo4IsglqzRFrTxIJ4tWc">
> Thanks, Pascale

Hi Pascale,
I found COL8A1 in one of the 67 annotations of collagen sheet (GO:0098646) from QuickGO, on page 3, with 1 record out of the 67 annotations.
<img width="1972" alt="Image" src="https://github.com/user-attachments/assets/a7b84b0b-2aa1-45c3-8745-0a710030da1b" />
I'm not sure why collagen sheet(GO:0098646) is not showing up in the parental list, looks like it's assigned by "PINC".

Best,
Daiqing

**Comment by pgaudet** [comment]
OK, I think it's because of the ontology: 
'collagen type VIII trimer' is a grand-child of collagen sheet (now called interstitial hexagonal collagen network), via its relation to sheet-forming collagen trimer

<img width="488" alt="image" src="https://github.com/user-attachments/assets/7c38d422-8064-4770-869c-8537b3069d41" />

It seems like something is not right there?

**Comment by DaiqingChen** [comment]
> OK, I think it's because of the ontology: 'collagen type VIII trimer' is a grand-child of collagen sheet (now called interstitial hexagonal collagen network), via its relation to sheet-forming collagen trimer
> 
> <img alt="image" width="488" src="https://private-user-images.githubusercontent.com/4782928/395635274-7c38d422-8064-4770-869c-8537b3069d41.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0NjU2MTIsIm5iZiI6MTczNDQ2NTMxMiwicGF0aCI6Ii80NzgyOTI4LzM5NTYzNTI3NC03YzM4ZDQyMi04MDY0LTQ3NzAtODY5Yy04NTM3YjMwNjlkNDEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTdUMTk1NTEyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NGYwM2MwZDcxMGFkYmU2N2EyZmEwYTM2OGNkYzdmYWE1Zjc1MzRiY2MxOTFhOGZhNmZhNDI2YTMwZGQxNGUwNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.7UxpIGM9lztDHRdJTnirLbRb83hFZXoNMEM7yPehw4s">
> It seems like something is not right there?

Yes, while we would like to obsolete term "sheet-forming collagen trimer(GO:0030935)", #29416

**Comment by DaiqingChen** [comment]
Hi @pgaudet ,
     I realized that the term definition is already up on QuickGO, thank you for the quick update! In the synonyms you put "collagen sheet" as exact, but it's wrong. Collagen sheet is the synonyms of collagen IV(#29401 ), not collagen X(this requested term). The reason I requested name change instead of obsoleting & creating new term is only because the annotation of the old term is almost the same(collagen X) as the new term. 
    Sorry for the misleading request, could you remove the synonym "collagen sheet"? Thanks!

**Comment by Matrisome** [comment]
@DaiqingChen : Please update with screenshot of revised slides 1 and 3

**Comment by pgaudet** [comment]
I think everything is done here. Please reopen if it wasn't.

**Comment by pgaudet** [comment]
Added 

[interstitial hexagonal collagen network](http://purl.obolibrary.org/obo/GO_0098646)
part_of GO:0005614 ! interstitial matrix
